src/OSPSuite.UI/Views/Charts/AxisSettingsView.cs

```diff
@@ -124,9 +123,14 @@ private RepositoryItem defaultColorRepository(Axis axis)
          return _colorRepository;
       }
 
-      private IGridViewColumn<Axis> createColumn<T>(Expression<Func<Axis, T>> propertyToBindTo, AxisOptionsColumns axisOptionsColumn, RepositoryItem repositoryItem = null, bool showInColumnChooser = true, string toolTip = null, Action<Axis> beforeNotificationAction = null)
+      private IGridViewColumn<Axis> createColumn<T>(Expression<Func<Axis, T>> propertyToBindTo,
+         AxisOptionsColumns axisOptionsColumn,
+         RepositoryItem repositoryItem = null,
+         bool showInColumnChooser = true,
+         string toolTip = null,
+         Action<Axis> beforeNotificationAction = null)
       {
-         var column = _gridBinderAxes.AutoBind(propertyToBindTo)
+         var column = _gridBinderAxes.Bind(propertyToBindTo)
```

Comment:
Same issue here/ AutoBind formats stuff on our behalf. Never realized that in PKSim because we are using commands anyways. In this case however, we write directly into the object => it resets the values